css/style.scss

```diff
@@ -194,6 +194,15 @@
 	margin-left: 5px;
 }
 
+#activity-personal-settings-link {
+	text-decoration: underline;
+	
+	a {
+		display: block;
+		padding: 10px 0px 10px 27px;
```

Comment:
Not a blocker for this PR but I see a lot of these magic numbers everywhere in Nextcloud codebase. Is there a plan to unify them a bit?  Personally, I like the way this is done in bootstrap/tailwind with css utilities like p-1, p-2, p-3, p-4, p-5

Reply:
hm... cannot answer that. I just put them in place so that the link is good positioned for this specific situation...

Reply:
> Is there a plan to unify them a bit? Personally, I like the way this is done in bootstrap/tailwind with css utilities like p-1, p-2, p-3, p-4, p-5

Would require further discussions with @nextcloud/server team :)

Reply:
did align it now to `padding: 10px 0px 10px 25px` which seems like the perfect values here...

Reply:
The general idea was more to move to shared vue components rather than implementing more global styles. For magic values we should slowly move more towards using css variables.

Reply:
Is it okay to merge this then or is there something I should change in this PR?

Reply:
@juliushaertl ok :) yeah shared Vue components make sense :)

@szaimen: Ship it :rocket: